bin/compile

```diff
@@ -18,7 +35,17 @@ cd $BUILD_DIR
 if [ -f shard.yml ]; then
   echo "-----> Installing Dependencies"
   $CRYSTAL_DIR/bin/crystal deps
+
+  if [ -f app.cr ]; then
+    echo "-----> Compiling app.cr (overriden - app.cr exists)"
+    $CRYSTAL_DIR/bin/crystal build app.cr --release
+  else
+    eval $(parse_yaml shard.yml "shard_")
```

Comment:
It does, but parses the whole file to that; I've left it as is so you can (maybe?) use some of the other variables. They'll end up as ``shard_{level 0 key}_{level 1 key}_....`` etc